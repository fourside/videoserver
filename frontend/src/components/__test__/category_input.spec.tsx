import * as React from 'react';
import 'jest';
import * as renderer from 'react-test-renderer';

import CategoryInput from '../category_input';

test('snapshot', () => {
  const props = {
    className: "",
    name: "",
    placeholder: "",
    onChange: () => {},
    onBlur: () => {},
    item: [""],
  };
  const component = renderer.create(<CategoryInput {...props} />).toJSON();
  expect(component).toMatchSnapshot();
});

