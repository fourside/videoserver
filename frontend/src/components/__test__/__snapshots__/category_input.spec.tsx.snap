// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<div
  aria-expanded={false}
  aria-haspopup="listbox"
  aria-labelledby="downshift-0-label"
  aria-owns={null}
  role="combobox"
>
  <input
    aria-activedescendant={null}
    aria-autocomplete="list"
    aria-controls={null}
    aria-labelledby="downshift-0-label"
    autoComplete="off"
    className=""
    id="downshift-0-input"
    name=""
    onBlur={[Function]}
    onChange={[Function]}
    onKeyDown={[Function]}
    placeholder=""
    value=""
  />
</div>
`;
